incubator/elasticsearch/README.md

```diff
@@ -1,7 +1,13 @@
 # Elasticsearch Helm Chart
 
-This chart is based on the [centerforopenscience/elasticsearch](https://hub.docker.com/r/centerforopenscience/elasticsearch/) image which comes with Fabric8's great [kubernetes discovery plugin](https://github.com/fabric8io/elasticsearch-cloud-kubernetes) for Elasticsearch.
- 
+This chart uses a standard Docker image of Elasticsearch (hub.docker.com/r/blacktop/elasticsearch) and uses a service pointing to the master's transport port for service discovery.
+Elasticsearch does not communicate with the Kubernetes API, hence no need for RBAC permissions.
+
+## Warning for previous users
```

Comment:
Are we certain this is tested thoroughly? If not, we might want to consider recommending people to create a new cluster instead. I had a (test) cluster fail during while I tested a very early version of these new changes, wouldn't want that to happen to anyone depending on this chart.